onnxruntime/core/optimizer/conv_bn_fusion.cc

```diff
@@ -124,24 +113,23 @@ Status ConvBNFusion::Apply(Graph& graph, Node& node, RewriteRuleEffect& rule_eff
   }
 
   // Replace initializers of conv node
-  graph.RemoveInitializedTensor(conv_W_tensor_proto->name());
+  graph_utils::ReplaceInitializer(graph, conv_W_tensor_proto->name(), new_conv_W_tensor_proto);
+
   if (conv_inputs.size() == 3) {
 #ifdef _MSC_VER
 #pragma warning(push)
 #pragma warning(disable : 6011)  // Not deferencing null pointer. conv_B_tensor_proto is set on line 93
```

Comment:
Is this hardcoded line still relevant?

Reply:
It is, as we're accessing conv_B_tensor_proto->name(), but it won't be after the next set of changes.

---
In reply to: [302298771](https://github.com/microsoft/onnxruntime/pull/1346#discussion_r302298771) [](ancestors = 302298771)